pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -310,6 +310,8 @@ func (c *Reconciler) reconcileDecider(ctx context.Context, pa *pav1alpha1.PodAut
 
 	// Ignore status when reconciling
 	desiredDecider.Status = decider.Status
+	resources.UpdateDecider(desiredDecider, config.FromContext(ctx).Autoscaler)
```

Comment:
I think this is a noop. The TickInterval should already be set by `MakeDecider` above.